tests/pytests/test_geometric.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from .utils import *
+
+import psi4
+from qcengine.testing import using
+
+
+@using('geometric')
+@pytest.mark.parametrize('opt', [
+    pytest.param({'engine' : 'optking'}, id='optking'),
```

Comment:
Dict's are likely not optimal here a simple `['optking', 'geometric']` is likely better.